srcpkgs/wayfire/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'wayfire'
+pkgname=wayfire
+version=0.1
+revision=1
+build_style=meson
+hostmakedepends="pkg-config cmake"
```

Comment:
Am I reading this right? It needs meson and cmake?

Reply:
you're right

Reply:
It shouldn't need cmake, are you sure it is needed? What error do you get without cmake?

Reply:
It was removed in the latest revision @ammen99